package.json

```diff
@@ -19,12 +18,12 @@
     "cookbook": "cd polyfill && npm install && npm run cookbook && cd ..",
     "test262": "cd polyfill && npm install && npm run test262",
     "lint": "eslint . --ext js,mjs",
-    "build:prepare": "mkdirp out && mkdirp out/docs",
+    "build:prepare": "mkdir -p out/docs || mkdir out\\docs || exit 0",
```

Comment:
this should still use mkdirp so that WSL is not a requirement. The slashes will work fine passed to any node script.

Reply:
hey @ljharb i checked the docs, and mkdir is alias for system-command md [1].  i don't think WSL comes into play ^^;;;

[1] md
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/md#BKMK_examples